stdlib/atomic.mli

```diff
@@ -17,6 +17,8 @@
 
 (** Atomic references.
 
+    See {%html:<a href="../parallelism.html">%}'Memory model: The hard bits' chapter{%latex:~\ref{c:memorymodel}%}{%html:</a>%} in the manual.
```

Comment:
This html link seems far too brittle to me. Typically, it would require rewriting when generating the manual for `ocaml.org`.